### DISCUSSION TITLE
[9주차][최단 경로] 회고 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/js-algorithm-basecamp#73] [General]
9주차 회고 글 모음

**Comment by bigsaigon333** [comment]
## 체크인

지금 기분을 10점 만점으로 표현해주세요~

**Reply by bigsaigon333** [reply]
8점: 친구들이랑 1박 2일로 놀러왔는데, 스터디 하려고 몰래 빠져나왔습니다~ :+1: :+1: :+1: :+1:

**Reply by ganeodolu** [reply]
7점 : 벌여놓은 일들이 계속 밀리네요 단디 마음을 잡는 것만으로는 부족한 것 같고, 뭔가 환경적으로 바꿔야될 것 같네요

**Comment by bigsaigon333** [comment]
## 체크아웃
나가기 전 기분을 10점 만점으로 표현해주세요~

**Reply by ganeodolu** [reply]
8점 : 버퍼 끝나고 간만에 모여서 근황얘기도 하고 좋았습니다. 근데 그래프 어렵네요 ㅜ.ㅜ
